app/renderer/components/autofill/autofillAddressPanel.js

```diff
@@ -45,118 +51,118 @@ class AutofillAddressPanel extends ImmutableComponent {
   onOrganizationChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('organization', e.target.value)
-    if (this.organization.value !== e.target.value) {
-      this.organization.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onStreetAddressChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('streetAddress', e.target.value)
-    if (this.streetAddress.value !== e.target.value) {
-      this.streetAddress.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onCityChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('city', e.target.value)
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
-    if (this.city.value !== e.target.value) {
-      this.city.value = e.target.value
-    }
   }
   onStateChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('state', e.target.value)
-    if (this.state.value !== e.target.value) {
-      this.state.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onPostalCodeChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('postalCode', e.target.value)
-    if (this.postalCode.value !== e.target.value) {
-      this.postalCode.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onCountryChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('country', e.target.value)
-    if (this.country.value !== e.target.value) {
-      this.country.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onPhoneChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('phone', e.target.value)
-    if (this.phone.value !== e.target.value) {
-      this.phone.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
   onEmailChange (e) {
     let currentDetail = this.props.currentDetail
     currentDetail = currentDetail.set('email', e.target.value)
-    if (this.email.value !== e.target.value) {
-      this.email.value = e.target.value
-    }
     windowActions.setAutofillAddressDetail(currentDetail, this.props.originalDetail)
   }
+
   onKeyDown (e) {
     switch (e.keyCode) {
       case KeyCodes.ENTER:
         this.onSave()
         break
       case KeyCodes.ESC:
-        this.props.onHide()
+        this.onHide()
         break
     }
   }
+
   onSave () {
-    appActions.addAutofillAddress(this.props.currentDetail, this.props.originalDetail)
```

Comment:
because this is this WIP and I need to check some things before finishing this PR